hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieInputFormatUtils.java

```diff
@@ -82,6 +82,13 @@
   public static final int HOODIE_PARTITION_PATH_COL_POS = 3;
   public static final String HOODIE_READ_COLUMNS_PROP = "hoodie.read.columns.set";
 
+  public static final String HOODIE_USE_META_FIELDS = "hoodie.use.meta.fields";
+  public static final String DEFAULT_HOODIE_USE_META_FIELDS = "true";
+  public static final String RECORD_KEY_FIELD = "hoodie.record.key.field";
```

Comment:
I think we should just reuse from `HoodieTableConfig` and not redefine all of these again